spec/unit/cli_spec.rb

```diff
@@ -219,6 +219,19 @@
         ])
       end
 
+      it "forwards the --remote-user option to the InspectTask" do
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
```

Comment:
Block argument expression is not on the same line as the block start.<br>Extra space before block parameter detected.